src/main/resources/interface-specification.yml

```diff
@@ -2871,16 +2936,46 @@ components:
           minimum: 0
         agreementApprovalPolicy:
           $ref: '#/components/schemas/AgreementApprovalPolicy'
-        activeDescriptor:
-          $ref: '#/components/schemas/CompactDescriptor'
+        eservice:
+          $ref: '#/components/schemas/CatalogEService'
+    CatalogEService:
+      type: object
+      required:
+        - id
+        - name
+        - description
+        - technology
+        - attributes
+        - descriptors
+        - isMine
+        - canSubscribe
+        - isSubscribed
+      properties:
+        id:
+          type: string
+          format: uuid
+        name:
+          type: string
+        description:
+          type: string
+        technology:
+          $ref: '#/components/schemas/EServiceTechnology'
+        attributes:
+          $ref: '#/components/schemas/EServiceAttributes'
+        descriptors:
+          type: array
+          items:
+            $ref: '#/components/schemas/CompactDescriptor'
```

Comment:
are other descriptors requested in this view?

Reply:
This field has been requested recently.